CHANGES.md

```diff
@@ -7,31 +7,32 @@ These endpoints have been added or updated since the last release.
-| [#254](https://github.com/ethereum/beacon-APIs/pull/254) add `finalized` to metadata for chain objects                                                                |                                                  |                                                   |                                                    |                                                 |                                           |
-| [#260](https://github.com/ethereum/beacon-APIs/pull/260) NEW `/eth/v1/beacon/rewards/blocks/{block_id}`                                                               |                                                  |                                                   |                                                    |                                                 |                                           |
-| [#260](https://github.com/ethereum/beacon-APIs/pull/260) NEW `/eth/v1/beacon/rewards/attestations/{epoch}`                                                            |                                                  |                                                   |                                                    |                                                 |                                           |
-| [#262](https://github.com/ethereum/beacon-APIs/pull/262) NEW `/eth/v1/beacon/rewards/sync_committee/{block_id}`                                                       |                                                  |                                                   |                                                    |                                                 |                                           |
+| Endpoint                                                                                                                                                             | [Lighthouse](https://github.com/sigp/lighthouse) | [Lodestar](https://github.com/ChainSafe/lodestar) | [Nimbus](https://github.com/status-im/nimbus-eth2) | [Prysm](https://github.com/prysmaticlabs/prysm) | [Teku](https://github.com/ConsenSys/teku) |
+|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------|---------------------------------------------------|----------------------------------------------------|-------------------------------------------------|-------------------------------------------|
+| [#290](https://github.com/ethereum/beacon-APIs/pull/290) add `el_offline` to `GetSyncingStatusResponse`                                                              |                                                  |                                                   |                                                    |                                                 |                                           |
```

Comment:
probably more meaningful to put 
`/eth/v1/node/syncing` added `el_offline` to response

Reply:
Done